notebooks/COS/Requirements.txt

```diff
@@ -0,0 +1,20 @@
+## Requirements
+python3.6
+numpy 
+matplotlib
+astropy
+astroquery
+glob
+sys
+os
+shutil
+datetime
+csv
```

Comment:
```suggestion
```
These are all in the python standard library so you don't need to put them in the `requirements` file.  requirements are only for 3rd-party packages.